hathor/manager.py

```diff
@@ -1007,6 +1008,11 @@ def on_new_tx(self, tx: BaseTransaction, *, conn: Optional[HathorProtocol] = Non
             self.tx_storage.indexes.mempool_tips.update(tx)  # XXX: move to indexes.update
         self.tx_fully_validated(tx, quiet=quiet)
 
+        if tx.is_block and (cast(Block, tx).get_height() % 25_000) == 0:
+            gcl = gc.get_objects()
+            gcl_blocks = [x for x in gcl if isinstance(x, Block)]
+            self.log.debug('blocks in memory', count=len(gcl_blocks))
+
```

Comment:
Should we gate this behind `__debug__`? Not sure if we use it, though

Reply:
`__debug__` is actually always `True` normally, only if running `python -O` it's turned of, but we never do that

Reply:
Removed.